src/styles/style.css

```diff
@@ -0,0 +1,163 @@
+.body {
+  margin: 0;
+
+  --border-color: rgb(240, 7, 228);
+  --hover-color: blue;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 199px;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  box-sizing: border-box;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin-top: 33px;
+  margin-left: 19px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  padding: 40px 16px 4px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  padding: 4px 16px;
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card:hover {
+  outline: #00acdc;
+  border: 1px solid var(--hover-color);
+  border-radius: 5px;
+  box-shadow: #ed0303;
```

Comment:
For the `.card:hover` selector, `outline` should be used with a style, width, and color (e.g., `outline: 1px solid #00acdc;`). Additionally, `box-shadow` requires values for offset, blur radius, and color (e.g., `box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);`).